doc/neps/nep-0048-spending-project-funds.rst

```diff
@@ -0,0 +1,421 @@
+"pay local market rates" and "same work -- same pay".
+
+We consider them both extreme:
+
+- "Same work -- same pay" is unfair to people living in locations with a higher
+  cost of living. For example, the average rent for a single family apartment
+  can differ by a large factor (from a few hundred dollar to thousands of
+  dollars per month).
+- "Pay local market rates" bakes in existing inequalities between countries
+  and makes fixed-cost items like a development machine or a holiday trip
+  abroad relatively harder to afford in locations where market rates are lower.
+
+We seek to find a middle ground between these two extremes.
+
+Useful points of reference include companies like GitLab and
+Buffer who are transparent about their remuneration policies ([3]_, [4]_),
+Google Summer of Code stipends ([5]_), other open source projects that manage
+their budget in a transparent manner (e.g., Babel and Webpack on Open
+Collective ([6]_, [7]_)), and standard salary comparison sites.
+
+Since NumPy is a not-for-profit project, we also looked to the nonprofit sector
+for guidelines on remuneration policies and compensation levels. Our findings
+show that most smaller non-profits tend to pay a median salary/wage. We
+recognize merit in this approach: applying candidates are likely to have a
+genuine interest in open source, rather than to be motivated purely by
+financial incentives.
+
+Considering all of the above, we will use the following guidelines for
+determining compensation:
+
+1. Aim to compensate people appropriately, up to a level that's expected for
+   senior engineers or other professionals as applicable.
+2. Establish a compensation cap of $125,000 USD that cannot be exceeded even
+   for the residents from the most expensive/competitive locations ([#f-pay]_).
+3. For equivalent work and seniority,  a pay differential between locations
+   should never be more than 2x.
+   For example, if we pay $110,000 USD to a senior-level developer from New
+   York, for equivalent work a senior-level developer from South-East Asia
+   should be paid at least $55,000 USD. To compare locations, we will use
+   `Numbeo Cost of Living calculator <https://www.numbeo.com/cost-of-living/>`__
+   (or its equivalent).
+
+Some other considerations:
+
+- Often, compensated work is offered for a limited amount of hours or fixed
+  term. In those cases, consider compensation equivalent to a remuneration
+  package that comes with permanent employment (e.g., one month of work should
+  be compensated by at most 1/12th of a full-year salary + benefits).
+- When comparing rates, an individual contractor should typically make 20% more
+  than someone who is employed since they have to take care of their benefits
+  and accounting on their own.
+- Some people may be happy with one-off payments towards a particular
+  deliverable (e.g., hiring a cleaner or some other service to use the saved
```

Comment:
It seems natural to read "hiring a cleaner" as an example "one-off payments toward a particular deliverable". That is not what is intended, though.

Reply:
Good point, that's a bit mixed up. I had two related but different things in mind:

1. A deliverable like "do a release" or "triage all open issues for label X", for a fixed price of $xxxx.
2. We've had cases where people motivated, for example, a NumFOCUS Small Development Grant with "I need X hours to complete proposed work Y". To free up that time, I'd like $2500 for 100 hours of time for my cleaner so I can justify working on Y.

Reply:
> 2. We've had cases where people motivated, for example, a NumFOCUS Small Development Grant with "I need X hours to complete proposed work Y". To free up that time, I'd like $2500 for 100 hours of time for my cleaner so I can justify working on Y.

It’s quite creative!:) However, I’m not convinced that such a corner case should be included in this document. I’ll leave the final decision up to you, @rgommers.

Reply:
If you want a concrete example to make the argument more compelling, you could mention childcare as an extra cost to be factored in.

Reply:
I like the childcare example too - seems good to keep.